src/background.js

```diff
@@ -359,20 +435,24 @@ const rollout = {
 
 const setup = {
   async start() {
-    const isAddonDisabled = await rollout.getSetting("doh-rollout.disable-heuristics", false);
-    const runAddon = await browser.experiments.preferences.getBoolPref("doh-rollout.enabled", false);
+    const isAddonDisabled = await rollout.getSetting(DOH_DISABLED_PREF, false);
```

Comment:
Note that this is a rather confusing name, since it seems to mean "DoH is disabled" but conflicts with the more generic 'addon is disabled'

Reply:
I think this comment may be out dated? Confirm that this is still an issue and I'm happy to revise it.

Reply:
Created issue just in case! #179